yarn.lock

```diff
@@ -1828,6 +1828,31 @@
   dependencies:
     "@ethersproject/logger" "^5.5.0"
 
+"@ethersproject/providers@5.4.0":
```

Comment:
Run `npx yarn-deduplicate` to remove the extra copy. (This should be it's own commit.)